axelrod/result_set.py

```diff
@@ -401,6 +423,13 @@ def _update_cooperation(self, p1, p2, cooperations):
         self.cooperation[p1][p2] += cooperations[0]
         self.cooperation[p2][p1] += cooperations[1]
 
+    def _update_state_distribution(self, p1, p2, counter):
+        self.state_distribution[p1][p2] += counter
```

Comment:
Added this and a bunch of others that were missing (whilst doing this I got annoyed at my prior lazy self).